plugins/amazonq/chat/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/amazonq/workspace/context/ProjectContextProviderTest.kt

```diff
@@ -57,4 +243,50 @@ class ProjectContextProviderTest {
         }
         verify(encoderServer, times(1)).encrypt(any())
     }
+
+    private fun createMockServer() = WireMockRule(wireMockConfig().dynamicPort())
 }
+
+val validQueryChatResponse = """
```

Comment:
can you reformat the JSON blobs
```suggestion
// language=JSON
val validQueryChatResponse = """
```

Reply:
@rli what does `// language=JSON` mean here

Reply:
isee, it tells IDE to treat it as JSON instead plain string